File: LoanManagementWebAPI/LoanManagementWebAPI/ViewModels/LoanDetails.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace LoanManagementWebAPI.ViewModels
{
    public class LoanDetails
    {
        public string ApplicantFirstName { get; set; }
        public string ApplicantLastName { get; set; }
        public string ApplicantAddress { get; set; }
        public int? CreatedBy { get; set; }
        public DateTime? CreatedDate { get; set; }
        public int? UpdatedBy { get; set; }
        public DateTime? UpdatedDate { get; set; }
    }
}

File: LoanManagementAngular/LoanManagementSystem/src/app/models/loandetails.ts
export class LoanDetails{
    loanId: number=0;
    applicantFirstName: string="";
    applicantLastName:string="";
    applicantAddress:string="";  
    loanType:string="";
    loanAmount:Number=0;
    loanTerm:string="";
    propertyAddress:string="";
    createdBy: Number=0;
    createdDate: Date = new Date();
    updatedBy: Number=0;
    updatedDate: Date=new Date();
}